.gitignore

```diff
@@ -23,3 +23,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# testing
+coverage/
```

Comment:
we will know the project's coverage.

![image](https://user-images.githubusercontent.com/7331511/193439544-ca9f35d1-6d4f-4545-af58-501e2e899ca5.png)